edumfa/lib/policy.py

```diff
@@ -727,37 +728,42 @@ def filter_policies_by_conditions(self, policies, user_object=None, request_head
         dbtoken = None
         for policy in policies:
             include_policy = True
-            for section, key, comparator, value, active in policy['conditions']:
+
+            #migrate to new condition format
```

Comment:
I'm not quite sure but that looks like that should be done in a real database migration.

Reply:
I think that this would be the best option but I am not sure how to implement this. Do you have a Idea where I could start?

Reply:
First you can use `./edumfa-manage db migrate` that should create a new migration file. Afterwards you can add the required logic